packages/steps/build.sh

```diff
@@ -11,6 +11,7 @@ if [[ "$npm_lifecycle_event" == "prepack" ]]; then
   echo 'Removing "dist_commonjs" and "dist_esm" folders...'
   rm -rf dist_commonjs dist_esm
 fi
+rm -rf dist_commonjs dist_esm
```

Comment:
Why did you need to add this? Can we remove it or submit a different pull request explaining the reason for this addition?